src/KnowledgeQuiz.tsx

```diff
@@ -31,6 +33,15 @@ type QuestionType = {
 type QuizAtomType = {
     id: string;
     questions: QuestionType[];
+    resultGroups: ResultGroupsType[];
+    sharingIcons?: JSX.Element;
```

Comment:
Yeah I was not initally keen on duplicating Icons, but its true it would simplify the props. Could I spin this out in another PR? As this logic needs to be changed for Personality Quiz as well

Reply:
I have removed `sharingIcons` logic